.github/workflows/BuildPR.yml

```diff
@@ -132,6 +132,26 @@ jobs:
         run: scripts/linux/buildUbuntu22and24.sh
       - name: Run tests
         run: ctest --test-dir build/qdlt
+  
+  packageUbuntu:
+    name: Build debian package for Ubuntu 24.04
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          submodules: true
+      - name: Install build environment
+        run: scripts/linux/installUbuntu22and24.sh
+      - name: Create package
+        run: ./build_viewer_debs_noble.sh
+      - name: Archive artifact
+        uses: actions/upload-artifact@v4
+        if: ${{ success() }}
+        with:
+          name: DLTViewer-Ubuntu-24.04
+          path: debtmp/covesa-dlt-viewer_*.deb
```

Comment:
Why not add these lines https://github.com/COVESA/dlt-viewer/pull/687/files#diff-74516a92922ffa904d2512757ae88933c5d93e319c9065f7d08058518bd2b607R147-R154 to existing 
```
buildUbuntu:
    name: ....
```
?